pkg/notes/maps/testdata/fullmap.yaml

```diff
@@ -0,0 +1,28 @@
+---
+pr:  123
```

Comment:
I think it would be useful to allow multiple mappings per file, which should work by:

```yaml
---
pr: 123
…
---
pr: 321
…
```
right?

Reply:
:+1:  On it.

Reply:
Done